code/extensions/FileSubsites.php

```diff
@@ -10,9 +24,9 @@ class FileSubsites extends DataExtension
     // considered 'global', unless set otherwise
     public static $default_root_folders_global = false;
 
-    private static $has_one=array(
-        'Subsite' => 'Subsite',
-    );
+    private static $has_one = [
```

Comment:
Not critical to upgrade, but it would be really nice if we could move all the common logic from these `*Subsites` extensions into a base `SubsitesExtension` class; FileSubsites would then only contain file-specific overrides to this.